contracts/colony/ColonyFunding.sol

```diff
@@ -205,6 +205,27 @@ contract ColonyFunding is ColonyStorage, PatriciaTreeProofs { // ignore-swc-123
     return (fundingPot.associatedType, fundingPot.associatedTypeId, fundingPot.payoutsWeCannotMake);
   }
 
+ function moveFundsBetweenPots(
+    uint256 _permissionDomainId,
+    uint256 _childSkillIndex,
+    uint256 _domainId,
+    uint256 _fromChildSkillIndex,
+    uint256 _toChildSkillIndex,
+    uint256 _fromPot,
+    uint256 _toPot,
+    uint256 _amount,
+    address _token
+  )
+  public
+  stoppable
+  authDomain(_permissionDomainId, _childSkillIndex, _domainId)
+  authDomain(_domainId, _fromChildSkillIndex, getDomainFromFundingPot(_fromPot))
+  authDomain(_domainId, _toChildSkillIndex, getDomainFromFundingPot(_toPot))
+  validFundingTransfer(_fromPot, _toPot)
+  {
+    moveFundsBetweenPotsFunctionality(_fromPot, _toPot, _amount, _token);
```

Comment:
I like this but I would prefer an approach which wrapped the unmodified `moveFundsBetweenPots`, so something along the lines of:

```
 function moveFundsBetweenPots(
    uint256 _permissionDomainId,
    uint256 _childSkillIndex,
    uint256 _domainId,
    uint256 _fromChildSkillIndex,
    uint256 _toChildSkillIndex,
    uint256 _fromPot,
    uint256 _toPot,
    uint256 _amount,
    address _token
  )
  public
  stoppable
  authDomain(_permissionDomainId, _childSkillIndex, _domainId)
{
  moveFundsBetweenPots(_domainId, _fromChildSkillIndex, _toChildSkillIndex, _fromPot, _toPot, _amount, _token);
}
```

That way the original function is left unchanged, and we have a special function for use in voting.

Reply:
If I'm understanding you correctly, don't think what you're proposing is possible, because of the reputation proofs. In the original function, the proofs prove the relationship between a domain where I have the permission, and the domains where we're moving funds from and to. In this new function, the reputation proofs prove the relationship between the domain where I'm saying I'm acting, and the domains we're moving the funds from and to. There is no universal way to convert from one to the other, and so we're not able to wrap them the way you're proposing.

I'm also very much in favour of deprecating that original function (indeed, I already have it marked as such in this PR), and switching to using the new `moveFundsBetweenPots` introduced here.